code/datums/ftl_ships/admin_ships.dm

```diff
@@ -49,3 +49,100 @@
 	recharge_rate = 200
 
 	init_components = list("2,3" = "engine")
+
+/datum/starship/ntescapeshuttle
+	name = "civillian escape shuttle"
+	faction = list("nanotrasen",0)
+	salvage_map = "nanotrasen_escape_shuttle"
+
+	x_num = 3
+	y_num = 3
+
+	hull_integrity = 5 // majority of the ship is hull, have fun living with the guilt of massacring civillians
+	shield_strength = 0 // no shields on this ride to hell
+	evasion_chance = 45 // decent evasion
+
+	repair_time = 50 // crew are desperately trying to survive
+	recharge_rate = 0
```

Comment:
It's not intended to be able to fight, which is why I've given it the flee operation